bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/internal/discovery/BlueZDiscoveryService.java

```diff
@@ -1,83 +0,0 @@
-/**
- * Copyright (c) 2010-2020 Contributors to the openHAB project
- *
- * See the NOTICE file(s) distributed with this work for additional
- * information.
- *
- * This program and the accompanying materials are made available under the
- * terms of the Eclipse Public License 2.0 which is available at
- * http://www.eclipse.org/legal/epl-2.0
- *
- * SPDX-License-Identifier: EPL-2.0
- */
-package org.openhab.binding.bluetooth.bluez.internal.discovery;
-
-import java.util.Collections;
-
-import org.eclipse.smarthome.config.discovery.AbstractDiscoveryService;
-import org.eclipse.smarthome.config.discovery.DiscoveryResult;
-import org.eclipse.smarthome.config.discovery.DiscoveryResultBuilder;
-import org.eclipse.smarthome.config.discovery.DiscoveryService;
-import org.eclipse.smarthome.core.thing.ThingUID;
-import org.openhab.binding.bluetooth.bluez.BlueZAdapterConstants;
-import org.osgi.service.component.annotations.Component;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import tinyb.BluetoothAdapter;
-import tinyb.BluetoothManager;
-
-/**
- * This is a discovery service, which checks whether we are running on a Linux with a BlueZ stack.
- * If this is the case, we create a bridge handler that provides Bluetooth access through BlueZ.
- *
- * @author Kai Kreuzer - Initial Contribution and API
- * @author Hilbrand Bouwkamp - Moved background scan to actual background method
- *
- */
-@Component(immediate = true, service = DiscoveryService.class, configurationPid = "discovery.bluetooth.bluez")
-public class BlueZDiscoveryService extends AbstractDiscoveryService {
```

Comment:
I suggest keeping this class. I'd prefer not to lose the automatic discovery feature of the current bluez binding.